### ISSUE TITLE
Menu with all available labels/status is displayed truncated on Firefox Mobile

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2158]
URL: https://staging.webcompat.com/issues/1378
Browser / Version: Firefox Nightly Mobile 60.0a1 (2018-02-18)
Operating System: Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Prerequisites:**
1. Github account available and signed in.

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/issues/1378
2. Scroll down the page to "Labels" and "Status" section.
3. Tap the "Edit" (labels/status) button. 

**Expected Behavior:**
Menu with all available labels/status is fully visible.

**Actual Behavior:**
Menu with all available labels/status is displayed truncated on the left side.

**Note**
1. Affected area:
```html
<div class="label-editor js-CategoryEditor">
	<div class="label-editor-header">
		<input class="form-field text-field" placeholder="Filter labels">
			<button class="nav-button js-CategoryEditor-close" tabindex="0">
				<svg class="nav-text icon nav-icon" viewBox="0 0 30 30" role="presentation">
					<title>Save &amp; close</title>
					<use href="#svg-sliders">Save &amp; close</use>
				</svg>
			</button>
		</div>
		<div class="label-editor-list" tabindex="-1">
			<label class="label-editor-list-item label-browser-android" tabindex="0">
				<input class="label-editor-list-item-checkbox" name="android" data-color="f7c6c7" data-remotename="browser-android" tabindex="-1" type="checkbox">
					<span class="small">android</span>
				</label>
				<label class="label-editor-list-item label-browser-chrome" tabindex="0">
					<input class="label-editor-list-item-checkbox" name="chrome" data-color="006b75" data-remotename="browser-chrome" tabindex="-1" type="checkbox">
						<span class="small">chrome</span>
					</label>
<!---not needed-->
	</div>
<div>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

![screenshot-1519049391060](https://user-images.githubusercontent.com/12184325/36381778-585f236e-158f-11e8-83d4-b4c0516d2ee5.jpg)

**Comment by softvision-oana-arbuzov** [comment]
Suggestion: Modifying `left` to `10px` to `min-width` to `390px`, the menu becomes fully visible.
```css
.label-editor {
    position: absolute;
    display: block;
    min-width: 320px;
    z-index: 5;
    white-space: nowrap;
    top: 13%;
    left: -49px;
    box-shadow: 0 1px 5px 0 rgba(0, 0, 0, .1);
    border-radius: 4px;
    background-color: rgba(255, 255, 255, 1);
}
```
![screenshot-1519050332120](https://user-images.githubusercontent.com/12184325/36382418-819729b4-1591-11e8-8ea5-eaf92d381cba.jpg)

**Comment by zoepage** [comment]
The positioning for the other breaking points are missing. This needs fixing. Thanks for reporting. :)

**Comment by zoepage** [comment]
merged #2133
